apps/server/src/modules/authentication/uc/account.uc.ts

```diff
@@ -313,6 +300,11 @@ export class AccountUc {
 		});
 	}
 
+	private async isAdmin(currentUser: ICurrentUser): Promise<boolean> {
```

Comment:
As written above, this method is not needed. Please check for permissions instead for roles.

Reply:
Same as above.